eip712/messages.py

```diff
@@ -138,9 +139,20 @@ def __getitem__(self, key: str) -> Any:
 
     @property
     def signable_message(self) -> SignableMessage:
-        """The current message as a :class:`SignableMessage` named tuple instance."""
+        """
+        The current message as a :class:`SignableMessage` named tuple instance.
+        **NOTE**: The 0x19 prefix is NOT included.
+        """
         return SignableMessage(
-            HexBytes(b"\x01"),
+            HexBytes(1),
             HexBytes(hash_domain(self._domain_)),
             HexBytes(hash_eip712_message(self._body_)),
         )
+
+    @property
+    def eip712_hash(self) -> HexBytes:
```

Comment:
If this could move to a pure function called `hash_message(msg: EIP712Message) -> HexBytes` I'd be happy with it